Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -18,6 +18,26 @@ import PackageLoading
 
 @testable import Build
 
+#if os(macOS)
+    let PLATFORM :String =  "x86_64-apple-macosx10.10"
```

Comment:
Please call this something like `defaultTargetTriple` (also adjust the position of the colon):

    let defaultTargetTriple: String =  "x86_64-apple-macosx10.10"